src/components/Restaurant/Restaurant.test.jsx

```diff
@@ -6,7 +6,7 @@ describe("Restaurant", () => {
   it("renders an Orders heading", () => {
     render(<Restaurant />);
     const titleElement = screen.getByRole("heading", {
-      level: 3,
+      name: 'Restaurant Orders',
```

Comment:
This is the same situation I commented on in the code above. I don't know which approach is better between the heading's content and the heading's level.